sprint0/sprint0/Bosses/Aquamentus/Aquamentus.cs

```diff
@@ -9,85 +9,84 @@
  */
 namespace sprint0
 {
-    public class Aquamentus : IEnemy
+    public class Aquamentus : AbstractEnemy
     {
-        private readonly Game1 game;
-        public Rectangle Location { get; set; }
-        public Texture2D Texture { get; set; }
-        public EnemyType Type { get => EnemyType.None; }
 
-        private readonly int xOffset = 1, yOffset = 11, width = 24, height = 32;
+        public EnemyType Type { get => EnemyType.None; }
+        private readonly int xOffset = 1, yOffset = 11;
         private readonly List<Rectangle> sources;
 
-        private int currFrame;
-        private readonly int totalFrames, repeatedFrames;
-
-        private Direction direction;
-        private int moveCount;
         private readonly int moveDelay = 5, minDistance = (int)(Game1.Width * Game1.Scale * 0.2), maxDistance = (int)(Game1.Width * Game1.Scale * 0.8);
-        private readonly bool isDead;
 
         private readonly int fireballRate = 100;
         private int fireballCounter = 0;
-        private int health;
-        public int Damage { get; }
-        private ItemSpawner itemSpawner;
         private int damageTimer = 0;
```

Comment:
we don't want new I don't think; can just use inherited fields / properites

Reply:
removed and are using the inherited ones